exercises/minesweeper/src/example.c

```diff
@@ -0,0 +1,66 @@
+#include "minesweeper.h"
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <ctype.h>
+
+static void set_single_field(char **minefield, const size_t row,
+                             const size_t col)
+{
+   if (minefield[row][col] == '*')
+      return;
+
+   const char c = minefield[row][col];
+   minefield[row][col] = (c == ' ') ? '1' : c + 1;
+}
+
+static void set_fields(char **minefield, const size_t all_rows,
+                       const size_t all_cols, const size_t row,
```

Comment:
Would `all_rows` and `all_cols` be better named as `row_count` and `col_count`? I believe that's how they are used here.